test/services/files/exports/export_table_to_file.service.test.js

```diff
@@ -60,10 +60,20 @@ describe('Export Table To File service', () => {
     await customer.$query().patch({ customerFileId })
 
     // We separately pull out the customer record so we can compare the exported file against the exact db contents
-    // including the createdAt and updatedAt fields
     customerRecord = await CustomerModel.query().findById(customer.id)
+
+    // Date fields are converted when exporting to a different format so update the record to the expected format
+    customerRecord.createdAt = formatDate(customerRecord.createdAt)
+    customerRecord.updatedAt = formatDate(customerRecord.updatedAt)
   })
 
+  // Formats a date by stringifying it, then removing the ""s from the resulting string
+  function formatDate (date) {
+    return JSON
+      .stringify(date)
+      .replace(/"/g, '')
+  }
```

Comment:
I have the opinion that any of these small helper methods we write should be added to `GeneralHelper`. That way we can keep track of what we've already got and possibly refactor rather than duplicate when an alternate is needed. What do you think?

Reply:
Yes makes sense!